addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -970,12 +985,23 @@ MockServer.include({
     _mockMailChannelMessagePost(id, kwargs, context) {
         const message_type = kwargs.message_type || 'notification';
         const channel = this._getRecords('mail.channel', [['id', '=', id]])[0];
-        if (channel.channel_type !== 'channel' && !channel.is_pinned) {
+        if (channel.channel_type !== 'channel') {
             // channel.partner not handled here for simplicity
             this._mockWrite('mail.channel', [
                 [channel.id],
-                { is_pinned: true },
+                {
+                    last_meaningful_action_time: datetime_to_str(new Date()),
+                },
             ]);
+            if (!channel.is_pinned) {
```

Comment:
this condition is not really necessary here, I think it's fine to write is pinned even if it is already true

Reply:
condition removed